src/qt/CMakeLists.txt

```
@@ -1,24 +1,21 @@
 if(IVW_APP_INVIWO OR IVW_APP_QTBASE)
 
-    # https://doc-snapshots.qt.io/qt6-dev/cmake-qt5-and-qt6-compatibility.html#supporting-older-qt-5-versions   
-    find_package(QT NAMES Qt6 Qt5 COMPONENTS Core QUIET)
-    find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core QUIET)
-
-    if(NOT Qt${QT_VERSION_MAJOR}Core_FOUND) 
+    find_package(Qt6 REQUIRED COMPONENTS Core QUIET)
+    if(NOT Qt6Core_FOUND) 
         message(FATAL_ERROR 
             "The inviwo app requiers Qt, we were not able to find it. Consider adding Qt to the "
             "CMAKE_PREFIX_PATH, i.e. start cmake with the option -DCMAKE_PREFIX_PATH=\"/path/to/qt\"")
     endif()
 
     # Create a qt.conf file in the same dir as the executable to help qt find its plugins
     # http://doc.qt.io/qt-5/qt-conf.html
```

Is this more correct now?
```suggestion
    # http://doc.qt.io/qt-6/qt-conf.html
```

---

mmm